frontend/public/components/graphs/gauge.jsx

```diff
@@ -159,14 +170,16 @@ class Gauge_ extends BaseGraph {
     const fontColor = fontColors[errorState];
 
     this.data[0].marker.colors[1] = color;
-    if (data < 1) {
+    if (data > 0 && data < 1) {
       data = data.toFixed(1);
     } else {
       data = Math.round(data);
     }
     this.layout.annotations[0].text = `${data}%`;
     this.layout.annotations[0].font.color = fontColor;
-    relayout(this.node, this.layout);
+    if (query) {
+      relayout(this.node, this.layout);
```

Comment:
It's not clear to me why we wouldn't relayout for non-query data

Reply:
When using `percentUsed` the relayout was causing problems, I don't recall exactly what it was, I think 'this.node' wasn't defined at the time the call was being made.

Reply:
Then we should return early when `this.node` is not defined (as long as this method is called again when it is).

Reply:
I was calling `updateGraph` in `base.jsx` too early, before `this.layout` was fully initialized. fixed.